astropy/coordinates/solar_system.py

```diff
@@ -0,0 +1,271 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+"""
+This module contains convenience functions for retrieving solar system
+ephemerides from jplephem.
+"""
+
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+
+from collections import OrderedDict
+import numpy as np
+from .sky_coordinate import SkyCoord
+from ..utils.data import download_file
+from .. import units as u
+from ..constants import c as speed_of_light
+from .representation import CartesianRepresentation
+from .builtin_frames import GCRS, ICRS
+from .builtin_frames.utils import get_jd12, cartrepr_from_matmul
+from .. import _erfa
+
+__all__ = ["get_body", "get_moon", "get_barycentric_body_position", "SOLAR_SYSTEM_BODIES"]
+
+KERNEL = None
+
+"""
+each value in the BODIES dictionary a list of kernel pairs needed
+to find the barycentric position of that object from the JPL kernel.
+"""
+SOLAR_SYSTEM_BODIES = ['sun', 'mercury', 'venus', 'earth-moon-barycenter',
+                       'earth', 'moon', 'mars', 'jupiter', 'saturn', 'uranus',
+                       'neptune', 'pluto']
+BODIES = OrderedDict(
+                    (('sun', [(0, 10)]),
+                     ('mercury', [(0, 1), (1, 199)]),
+                     ('venus', [(0, 2), (2, 299)]),
+                     ('earth-moon-barycenter', [(0, 3)]),
+                     ('earth',  [(0, 3), (3, 399)]),
+                     ('moon', [(0, 3), (3, 301)]),
+                     ('mars', [(0, 4)]),
+                     ('jupiter', [(0, 5)]),
+                     ('saturn', [(0, 6)]),
+                     ('uranus', [(0, 7)]),
+                     ('neptune', [(0, 8)]),
+                     ('pluto', [(0, 9)]))
+                     )
+SOLAR_SYSTEM_BODIES = list(BODIES.keys())
```

Comment:
Oops, I just realized the here you just did what was suggested by @mhvk above.  See my suggestion there that this should be a ``tuple`` rather than list.  And you should also remove the ``SOLAR_SYSTEM_BODIES`` above.